presto-raptor/src/main/java/com/facebook/presto/raptor/storage/ShardRecoveryManager.java

```diff
@@ -142,7 +142,9 @@ private void enqueueMissingShards()
     {
         missingShardExecutor.scheduleWithFixedDelay(() -> {
             try {
-                SECONDS.sleep(ThreadLocalRandom.current().nextInt(1, 30));
+                // Jitter to avoid overloading database
```

Comment:
Nit: this should be lowercase. We only use uppercase for comments if they are complete sentences.